src/modules/ie-objects/ObjectDetailPage.tsx

```diff
@@ -725,15 +741,23 @@ export const ObjectDetailPage: FC<DefaultSeoInfo> = ({ title, description, image
 		(format: MetadataExportFormats) => {
 			switch (format) {
 				case MetadataExportFormats.fullNewspaperZip:
-					window.open(
-						`${publicRuntimeConfig.PROXY_URL}/${NEWSPAPERS_SERVICE_BASE_URL}/${ieObjectId}/${IE_OBJECTS_SERVICE_EXPORT}/zip`
-					);
+					setCopyrightModalOpen(true);
+					setOnConfirmCopyright(() => () => {
+						window.open(
+							`${publicRuntimeConfig.PROXY_URL}/${NEWSPAPERS_SERVICE_BASE_URL}/${ieObjectId}/${IE_OBJECTS_SERVICE_EXPORT}/zip`
+						);
+						handleOnDownloadEvent();
+					});
```

Comment:
OPTIONAL: preferably we would have:
* user clicks button => handleExportButtonClicked
* exportButtonClicked => setConfirmModalOpen(true)
* modal is shown
* modal has confirmCallback => handleExport()
* handleExport does the window.open(exportLink) + trigger event